crates/rooch/src/commands/move_cli/commands/publish.rs

```diff
@@ -48,6 +48,10 @@ pub struct Publish {
     /// `moveos_std::account_storage::publish_modules_entry`
     #[clap(long, parse(from_flag))]
     pub by_move_action: bool,
+
+    /// Whether a password should be provided
+    #[clap(short = 'p', long = "password")]
+    password_required: Option<bool>,
```

Comment:
Is it possible to auto-detect that if the private key is encrypted, then prompt the password input?

Reply:
The private key is always encrypted. When the user does not enter the password, the password defaults to "".